To-do-list/Controller/TaskViewController.swift

```diff
@@ -45,12 +46,8 @@ class TaskViewController: UIViewController, UITextFieldDelegate, UITextViewDeleg
     @IBAction func saveButtonTapped(_ sender: Any) {
         guard let title = titleField.text, !title.isEmpty else { return }
         guard let description = descriptionField.text, !description.isEmpty else { return }
-        
-        let taskDictionnary = ["title": title, "description": description]
-        UserDefaults.standard.set(taskDictionnary, forKey: "newTask")
-        
-        completionHandler?()
-        navigationController?.popToRootViewController(animated: true)
+        taskViewModel?.userDefaultsRefresh(title: title, description: description)
+        taskViewModel?.refreshMainViewController()
```

Comment:
let's create a function like viewModel.saveTask(title: title, description: description)
instead of 49-50 lines

Reply:
Yes, created the func in taskViewModel

Reply:
How will I return to rootController without line 50? This goes by appDelegate